sql/mob_pools.sql

```diff
@@ -2233,7 +2233,7 @@ INSERT INTO `mob_pools` VALUES (2177,'Kadjaya','Kadjaya',149,0x01000B02000087208
 INSERT INTO `mob_pools` VALUES (2178,'Kadraeth_the_Hatespawn','Kadraeth_the_Hatespawn',233,0x0000EF0600000000000000000000000000000000,4,4,12,240,100,0,0,0,0,2,0,32,0,157,0,0,252,0,0,920,233);
 INSERT INTO `mob_pools` VALUES (2179,'Kagetora','Kagetora',149,0x01000E013B10042058305840585085613A711E80,13,13,9,240,100,0,0,0,0,0,0,32,0,153,0,0,7,0,0,149,149);
 INSERT INTO `mob_pools` VALUES (2180,'Kaiser_Behemoth_S','Kaiser_Behemoth',479,0x0000940100000000000000000000000000000000,1,1,6,380,100,1024,1,1,1,18,0,32,307,155,0,0,126,0,0,479,479);
-INSERT INTO `mob_pools` VALUES (2181,'Kalamainu','Kalamainu',174,0x0000490100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,18,1,32,6623,157,0,0,0,0,0,711,174);
+INSERT INTO `mob_pools` VALUES (2181,'Kalamainu','Kalamainu',174,0x0000490100000000000000000000000000000000,1,1,7,150,100,0,1,0,1,18,1,32,6623,157,0,0,0,0,0,711,174);
```

Comment:
I will need to see some evidence that they are 150 delay attack speeds beyond eyballing a video. If gleaned from video, I want to know the method 150 was determined. While possible historically it has not been reliable.

Reply:
Yeah I simply lined up the attack rate with videos. it was significantly higher than 150. I can leave it at default if that's prefered.

Reply:
yes. I the vid the lizard was getting approximately 2 attacks to that great axe warriors 1 (double attacks excluded) and since great axes are typically significantly over 300 I can be reasonably sure it's more than 150, even if that doesn't pin down what its precise attack speed is. better to leave alone when we don't have a new number we can prove.